### PR TITLE
Update Nations_Italian.json

### PR DESCRIPTION
**Pull request by Smashfanful**

### CODE REVIEW

**Comment by yairm210** [comment]
Build is failing

**Comment by Smashfanful** [comment]
How?

File: android/assets/jsons/Nations/Nations_Italian.json [hunk 1]
```diff
@@ -182,7 +189,8 @@
 		"unique": "+2 Cultura al turno dalle città prima della scoperta dell'Energia a Vapore",
 		"cities": ["Parigi","Orleans","Lione","Troyes","Tours","Marsiglia","Chartres","Avignon","Rouen","Grenoble",
 			"Dijon","Amiens","Cherbourg","Poitiers","Tolosa","Bayonne","Strasburgo","Brest","Bordeaux","Rennes",
-			"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen"]
+			"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen"]				"Nizza","Saint Etienne","Nantes","Reims","Le Mans","Montpellier","Limoges","Nancy","Lilla","Caen","Toulon",
```
Comment (r3versi) [inline android/assets/jsons/Nations/Nations_Italian.json]: This line. There is an extra ] after Caen

Comment (Smashfanful) [reply android/assets/jsons/Nations/Nations_Italian.json]: Okay, corrected

**Comment by Smashfanful** [comment]
Ok, now that should work

**Comment by yairm210** [comment]
With the nations now being included in the translation file, this is no longer relevant =)

**Comment by JackRainy** [comment]
Some of the translations are still relevant, I will assist in transfering them.

**Pull request closed without merging**